tasks/request_files/request_files.py

```diff
@@ -138,19 +137,26 @@ def task(
         )
         retry_sleep_secs = DEFAULT_RESTORE_RETRY_SLEEP_SECS
 
-    # Get retrieval type
-    try:
-        retrieval_type = os.environ[OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY]
-        if retrieval_type not in ("Standard", "Bulk", "Expedited"):
-            msg = (
-                f"Invalid RESTORE_RETRIEVAL_TYPE: '{retrieval_type}'"
-                " defaulting to 'Standard'"
-            )
-            LOGGER.info(msg)
-            retrieval_type = DEFAULT_RESTORE_RETRIEVAL_TYPE
-    except KeyError:
-        LOGGER.warn("Invalid RESTORE_RETRIEVAL_TYPE: 'None' defaulting to 'Standard'")
+    VALID_RESTORE_TYPES = ["Bulk", "Expedited", "Standard"] 
+
+    # Get config and env set to None if the key does not exist
+    env_retrieval_type = os.getenv(OS_ENVIRON_RESTORE_RETRIEVAL_TYPE_KEY, None)
```

Comment:
Update OS_ENVIRON key and key name to include DEFAULT

Reply:
updated

Reply:
Remember to revisit once naming standards are fixed up.